Frontend/CO.CDP.OrganisationApp/ThirdPartyApiClients/CompaniesHouse/CompaniesHouse.cs

```diff
@@ -0,0 +1,70 @@
+using Flurl;
+using Flurl.Http;
+
+namespace CO.CDP.OrganisationApp.ThirdPartyApiClients.CompaniesHouse;
+
+public class CompaniesHouseApi(IConfiguration configuration, ILogger<CompaniesHouseApi> logger) : ICompaniesHouseApi
```

Comment:
Can we have unit test for the class?

Reply:
Will add to ticket [DP-614](https://noticingsystem.atlassian.net/jira/software/projects/DP/boards/22?selectedIssue=DP-614)